bazel/mkl_dnn.BUILD

```diff
@@ -40,24 +56,31 @@ config_setting(
 # TODO(agramesh1) Automatically get the version numbers from CMakeLists.txt.
 
 template_rule(
-    name = "mkldnn_version_h",
-    src = "include/mkldnn_version.h.in",
-    out = "include/mkldnn_version.h",
+    name = "dnnl_version_h",
+    src = "include/dnnl_version.h.in",
+    out = "include/dnnl_version.h",
     substitutions = {
-        "@MKLDNN_VERSION_MAJOR@": "0",
-        "@MKLDNN_VERSION_MINOR@": "21",
-        "@MKLDNN_VERSION_PATCH@": "3",
-        "@MKLDNN_VERSION_HASH@": "N/A",
+        "@DNNL_VERSION_MAJOR@": "1",
+        "@DNNL_VERSION_MINOR@": "4",
+        "@DNNL_VERSION_PATCH@": "0",
+        "@DNNL_VERSION_HASH@": "N/A",
     },
 )
 
+
 cc_library(
     name = "mkl_dnn",
     srcs = glob([
         "src/common/*.cpp",
         "src/common/*.hpp",
         "src/cpu/*.cpp",
         "src/cpu/*.hpp",
+        "src/cpu/jit_utils/*.cpp",
```

Comment:
Question regarding changes for bazel build. Are these changes being tested anywhere? Since we removed bazel build from being tested on ci.

Reply:
No, these aren't tested. I wasn't sure what we want to do with the Bazel files wrt upgrading nGraph version. Since there were already changes in the existing PR, I decided to let them be.

Reply:
If it is not being tested we should not be upgrading the bazel build as part of the PR. 
The bazel build can be upgraded in a separate PR when we are able to build with latest mkl version. We can make it part of ci then.

Reply:
Done